frontend/src/components/CookingTime.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from "react";
+import { Button } from "@mui/material";
+
+import { API_URL } from "../utils/urls";
+
+const CookingTime = (props) => {
+  // eslint-disable-next-line
+  const [recipe, setRecipe] = useState([]);
+
+  useEffect(() => {
+    fetch(API_URL("recipes"))
+      .then((res) => res.json())
+      .then((json) => {
+        setRecipe(json);
+      });
+  }, [props.recipeCookingTime, props.recipeId]);
+
+  return (
+    <>
+      <Button
+        variant="outlined"
+        size="small"
+        sx={{
+          mr: 1,
+          mb: 1,
+        }}
+        onClick={() => {
+          props.onRecipeCookingTimeFilterClick(props.recipeCookingTime);
+        }}
+      >
+        {props.recipeCookingTime}
+      </Button>
+    </>
```

Comment:
Same case here. Morever, `CookingTime.js` and `Category.js` components seems to be almost the same, maybe you could make one component from them?